models/pulsepacket_generator.cpp

```diff
@@ -220,7 +220,7 @@ nest::pulsepacket_generator::update( Time const& T, const long from, const long
     long prev_spike = B_.spiketimes_.front();
     B_.spiketimes_.pop_front();
 
-    if ( n_spikes > 0 && prev_spike != B_.spiketimes_.front() )
+    if ( n_spikes > 0 && ( not B_.spiketimes_.empty() && prev_spike != B_.spiketimes_.front() ) )
```

Comment:
Please replace `&&` with `and` according to our current style guide. I also think that the extra `()` are not needed since we have a sequence of ands.